src/status_im/contexts/settings/wallet/events_test.cljs

```diff
@@ -57,23 +57,23 @@
                     (sut/remove-keypair cofx [mock-key-uid])))))))
 
 (deftest make-keypairs-accounts-fully-operable-test
-  (let [db                 (mock-db [{:key-uid            mock-key-uid
-                                      :lowest-operability :no
-                                      :accounts           [{:key-uid mock-key-uid :operable "no"}]}]
-                                    {"0x1" {:key-uid mock-key-uid :operable "no"}})
+  (let [db                 (mock-db {mock-key-uid {:key-uid            mock-key-uid
+                                                   :lowest-operability :no
+                                                   :accounts           [{:key-uid  mock-key-uid
+                                                                         :operable :no}]}}
+                                    {"0x1" {:key-uid mock-key-uid :operable :no}})
         key-uids-to-update [mock-key-uid]]
     (testing "make-keypairs-accounts-fully-operable"
       (let [effects         (sut/make-keypairs-accounts-fully-operable {:db db} [key-uids-to-update])
             result-db       (:db effects)
-            updated-keypair (some #(when (= (:key-uid %) mock-key-uid) %)
-                                  (get-in result-db [:wallet :keypairs]))
+            updated-keypair (get-in result-db [:wallet :keypairs mock-key-uid])
             updated-account (get-in result-db [:wallet :accounts "0x1"])]
-        (is (= (keyword (-> updated-keypair :accounts first :operable)) :fully))
-        (is (= (keyword (:operable updated-account)) :fully))
+        (is (= (-> updated-keypair :accounts first :operable) :fully))
+        (is (= (:operable updated-account) :fully))
         (is (= (:lowest-operability updated-keypair) :fully))))))
 
 (deftest connection-string-for-import-keypair-test
-  (let [cofx              {:db (mock-db [] {})}
+  (let [cofx              {:db (mock-db {} {})}
```

Comment:
No suggestion for change since this was already here before this PR, but the word `mock` in `mock-db` is imprecise IMO because there's no behavior being mocked. The db is just a test db. A mock is a different beast, it's usually meant to create fake instances that often return nil on every method/function call, unless the method/function is explicitly mocked.

Reply:
I will remove this function as it's only used three times in this namespace and out of three, we pass only an empty map to two instances.